.travis.yml

```diff
@@ -14,7 +11,7 @@ before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cp -r $TRAVIS_BUILD_DIR/* /Volumes/build ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cd /Volumes/build ; fi
 install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install binutils ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gnu-sed --with-default-names ; fi
 script:
-  - npm install
+  - npm install --ignore-scripts
```

Comment:
The `--ignore-scripts` flag should not be needed now, it was mostly a hack. Without it, It could be downloaded by default a prebuild image, but doesn't hurt too much beyond that.